crown-court-contribution/src/main/java/uk/gov/justice/laa/crime/contribution/exeption/CCCExceptionHandler.java

```diff
@@ -0,0 +1,30 @@
+package uk.gov.justice.laa.crime.contribution.exeption;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+import uk.gov.justice.laa.crime.commons.exception.APIClientException;
+import uk.gov.justice.laa.crime.contribution.dto.ErrorDTO;
+
+@Slf4j
+@RestControllerAdvice
+public class CCCExceptionHandler {
+
+    private static ResponseEntity<ErrorDTO> buildErrorResponse(HttpStatus status, String message) {
+        return new ResponseEntity<>(ErrorDTO.builder().code(status.toString()).message(message).build(), status);
+    }
+
+    @ExceptionHandler(ValidationException.class)
+    public ResponseEntity<ErrorDTO> handleValidationException(ValidationException exception) {
+        log.error("Validation exception: {}", exception);
```

Comment:
The slf4j loggers can sometimes lead you into a false sense of security ;)

Here because of method overloading you are actually calling the below method on `log`

```
    /**
     * Log an exception (throwable) at the ERROR level with an
     * accompanying message.
     *
     * @param msg the message accompanying the exception
     * @param t   the exception (throwable) to log
     */
    public void error(String msg, Throwable t);
```

instead of this one (which I presume you intended), so your log would actually look print exactly like `Validation exception: {}` following by the exception message and stack-trace of the exception

```
    /**
     * Log a message at the ERROR level according to the specified format
     * and argument.
     * 
     * <p>This form avoids superfluous object creation when the logger
     * is disabled for the ERROR level. 
     *
     * @param format the format string
     * @param arg    the argument
     */
    public void error(String format, Object arg);
```

You could try one of the following:

```
log.error("Validation exception: %s".formatted(exception.getMessage()), exception);
log.error("Validation exception: {}", exception.getMessage());
```